configs/Toxic_comment_classification_ModernBERT.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "Jigsaw_ModernBERT",
+    "n_gpu": 1,
+    "batch_size": 10,
+    "accumulate_grad_batches": 3,
+    "loss": "binary_cross_entropy",
+    "arch": {
+        "type": "ModernBERT",
+        "args": {
+            "num_classes": 6,
+            "model_type": "answerdotai/ModernBERT-base",
+            "model_name": "ModernBertForSequenceClassification",
+            "tokenizer_name": "AutoTokenizer"
+        }
+    },
+    "dataset": {
+        "type": "JigsawDataOriginal",
+        "args": {
+            "train_csv_file": "jigsaw_data/jigsaw-toxic-comment-classification-challenge/train.csv",
+            "test_csv_file": "jigsaw_data/jigsaw-toxic-comment-classification-challenge/val.csv",
+            "add_test_labels": false,
+            "classes": [
+                "toxicity",
+                "severe_toxicity",
+                "obscene",
+                "threat",
+                "insult",
+                "identity_attack"
+            ]
+        }
+    },
+    "optimizer": {
+        "type": "Adam",
```

Comment:
Might be worth using the hyperparameters given here:

https://github.com/AnswerDotAI/ModernBERT/blob/main/examples/finetune_modernbert_on_glue.ipynb

```
    "optimizer": {
        "type": "AdamW",
        "args": {
            "lr": 8e-5,
            "weight_decay": 8e-6,
            "betas": [0.9, 0.98],
            "eps": 1e-6,
            "amsgrad": false
        }
    }
```